src/content/1.7/development/webservice/_index.md

```diff
@@ -195,6 +195,61 @@ Here is what a resource API call could look like (in this case `http://example.c
 </prestashop>
 ```
 
+#### Available parameters
+
+You can add these **GET** parameters to your request to modify the READ response:
+
+* `display` to control which fields are returned
+* `filter` to control which items are returned
+* `language` to control which language values are returned
+
+##### Control returned fields with "display"
+
+The `display` parameter can be used to return all fields when used with the **full** value: `http://example.com/api/addresses/1?display=full`.
+
+You can also ask for certain fields if you use a list of fields names in brackets: `http://example.com/api/addresses/1?display=[id,lastname,firstname,phone_mobile]`
+
+{{% notice note %}}
+A response obtained with "display" other than "full" can't be used in a **PUT** (update) request, because the `WebserviceRequest` class validation for fields is the same for **POST** (create) and **PUT** (update).
+
+This should be fixed in a near future with a *yet-to-come-pull-request*!
+{{% /notice %}}
+
+##### Control returned items with "filter"
+
+The **EQUAL** operator is used when you need to get specific items. For exemple, if you want the addresses for customer #1, you can filter your **GET** request with the `filter` parameter: `http://example.com/api/addresses?filter[id_customer]=1`
+
+The **LIKE** operator is used when you need to search for items. For exemple, if you want the addresses with cities starting with "SAINT": `http://example.com/api/addresses?filter[city]=[saint]%`
+
+The **AND** operator is used when you need to get items matching several criteria: `http://example.com/api/addresses?filter[city]=[paris|lyon]`
+
+Other operators can be used, such as: 
+
+- **NOT EQUAL** (single value): `http://example.com/api/customers?filter[firstname]=![hubert]` (apologies to all Huberts)
+- **NOT EQUAL** (multiple values): `http://example.com/api/customers?filter[firstname]=![hubert|leon|gaspard]` (apologies again...)
+- **GREATER THAN**: `http://example.com/api/customers?filter[birthday]=>[2000-00-00%2000:00:00]` (millenials only :-))
```

Comment:
```suggestion
- **GREATER THAN OR EQUAL**: `http://example.com/api/customers?filter[birthday]=>[2000-00-00%2000:00:00]` (millenials only :-))
```

Reply:
Hi! I'm pretty sure it's "GREATER THAN" only: https://github.com/PrestaShop/PrestaShop/blob/develop/classes/webservice/WebserviceRequest.php#L1722

Reply:
He's right, it's "greater than" only.
Because the filter is `filter[birthday]` = `>[2000-00-00%2000:00:00]` you missread because of `=` assignment :)

Reply:
Thanks. It's a misreading.